vault/resource_azure_secret_backend_role.go

```diff
@@ -222,32 +254,43 @@ func azureSecretBackendRoleRead(_ context.Context, d *schema.ResourceData, meta
 		return nil
 	}
 
-	for _, k := range []string{
-		"ttl",
-		"max_ttl",
-		"application_object_id",
-		"permanently_delete",
-	} {
+	for _, k := range azureSecretFields {
 		if v, ok := resp.Data[k]; ok {
 			if err := d.Set(k, v); err != nil {
 				return diag.FromErr(fmt.Errorf("error reading %s for Azure Secret role Backend Role %q: %q", k, path, err))
 			}
 		}
 	}
 
-	if v, ok := resp.Data["azure_roles"]; ok {
+	if v, ok := resp.Data[consts.FieldPermanentlyDelete]; ok {
+		if err := d.Set(consts.FieldPermanentlyDelete, v); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting permanently delete field: %s", err))
```

Comment:
nit: I think this can just be
```suggestion
			return diag.Errorf("error setting permanently delete field: %s", err)
```

Reply:
Thank you! Just updated it! 👍